### DISCUSSION TITLE
Mosh Command using logical OR (||) not working

### DISCUSSION THREAD

**Discussion by simon-bueler** [blinksh/blink#2005] [Ideas]
It is a fairly wide used practice to have mosh start tmux. Mostly something like this is used:
```
tmux a || tmux
```
This simply attaches to an existing tmux session or (||) starts a new one when no session is available.
Unfortunatetly this is not supported by blink and it would be nice if it would be.

**Comment by simon-bueler** [comment]
At the current version it creates always a new session and never attaches to an existing one.

**Reply by carloscabanero** [reply]
There is (I think) -A, that would attach or create a new session, so you don't really need the || part.

How are you running the command though, are you doing `mosh server -- tmux -a || tmux`, something like that? Or are you passing it as part of Blink configuration?
